nengo_gui/components/pointer.py

```diff
@@ -53,11 +69,12 @@ def gather_data(self, t, x):
         msg = '%g %s' % (t, text)
         self.data.append(msg)
         if self.override_target is None:
-            return self.vocab_in.parse('0').v
+            return self.default_override_val
         else:
             v = (self.override_target.v - x) * 3
-            if self.vocab_in is not self.vocab_out:
-                v = np.dot(self.vocab_out.transform_to(self.vocab_in), v)
+            if self.loop_in:
+                if self.vocab_in is not self.vocab_out:
+                    v = np.dot(self.vocab_out.transform_to(self.vocab_in), v)
```

Comment:
That call to ```transform_to``` is going to be incredibly expensive to compute every timestep.  If we do something like this, it'd be a huge speed improvement to do it outside the loop, I think.

I'm also unsure that we even need this complexity.  The one case where it could be used is an AssociativeMemory with a different vocab for the input and output, and I'd be tempted to just remove that case from the whitelist.